From a74aa84d1c8307a298567d73d2785354e7175c18 Mon Sep 17 00:00:00 2001
From: pad <yoann.padioleau@gmail.com>
Date: Thu, 11 Aug 2016 11:48:14 -0700
Subject: [PATCH] * external/ocamlzip/zip.ml: remove duplicate exception. It is
 causing compilation error with ocaml 4.02.3 apparently. Should fix issue #133
 and issue #145

---
 external/ocamlzip/zip.ml | 2 --
 1 file changed, 2 deletions(-)

diff --git a/external/ocamlzip/zip.ml b/external/ocamlzip/zip.ml
index f49b91a..b6ad634 100644
--- a/external/ocamlzip/zip.ml
+++ b/external/ocamlzip/zip.ml
@@ -73,8 +73,6 @@ type out_file =
     mutable of_entries: entry list;
     of_comment: string }
 
-exception Error of string * string * string
-
 (* Return the position of the last occurrence of s1 in s2, or -1 if not
    found. *)
 
-- 
2.9.2

